Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -93,6 +95,7 @@ public class CarPlayManager: NSObject {
             locationProvider.stopUpdatingHeading()
             if let passiveLocationProvider = locationProvider as? PassiveLocationProvider {
                 passiveLocationProvider.locationManager.pauseTripSession()
+                os_log("Trip session paused", log: logger, type: .debug)
```

Comment:
It'd be good to go through all methods in `CarPlayManager` and see what CarPlay related methods it uses. I think that we should log all callbacks we receive from it.